cmd/src/server.rs

```diff
@@ -324,9 +324,13 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     let mut status_server = StatusServer::new(server_cfg.status_thread_pool_size, cfg.clone());
     if status_enabled {
         // Start the status server.
-        if let Err(e) = status_server.start(server_cfg.status_addr) {
+        if let Err(e) = if cfg.security.cert_path.is_empty() && cfg.security.key_path.is_empty() {
```

Comment:
What if only one of `cert_path` or `key_path` is empty?

Reply:
It cannot pass the configuration validation.